CODEOWNERS

```
@@ -1,21 +1,12 @@
-# These owners will be the default owners for everything in
-# the repo. Unless a later match takes precedence,
-# @mendersoftware/backend-dependabot-reviewers
-# and @mendersoftware/frontend-dependabot-reviewers will be requested for
-# review when someone opens a pull request.
-* @mendersoftware/backend-dependabot-reviewers @mendersoftware/frontend-dependabot-reviewers
+# Dependency updates
 
-# Order is important; the last matching pattern takes the most
-# precedence.
+go.mod @mendersoftware/backend-dependabot-reviewers
+go.sum @mendersoftware/backend-dependabot-reviewers
+_requirements_.txt @mendersoftware/backend-dependabot-reviewers
```

Shouldn't it be:
```suggestion
*requirements_.txt @mendersoftware/backend-dependabot-reviewers
```
?

---

indeed! overly eager auto formatting I guess... - should now be fixed 👌